.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying. -->
+
+
+## What is the new behavior?
+<!-- Please describe the behavior or changes that are being added by this PR. -->
+
+-
+-
+-
+
+---
+
+## Pull request checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tests(`npm run test`) for the changes have been added (for bug fixes / features) and pass
+- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
+- [ ] Build (`npm run build`) was run locally and any changes were pushed
+- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
```

Comment:
Let's change the first and last of these to `make test` and `make lint` (I'll create a Makefile today) and remove the third.